role/api.py

```diff
@@ -0,0 +1,50 @@
+"""
+Functions for handling roles
+"""
+
+from __future__ import unicode_literals
+
+from django.contrib.auth.models import Group
+from guardian.shortcuts import assign_perm
+
+from role.permissions import RepoPermission, GroupTypes
+
+
+def roles_init_new_repo(repo):
+    """
+    Create new groups for the repo
+    """
+    administrator_group = Group.objects.get_or_create(
```

Comment:
I love [this](https://groups.google.com/forum/#!msg/django-users/AKpy31__Nec/vW0eLf42COcJ) `django-users` post. I keep it bookmarked for times like this.